lib/parsers/log-parser.js

```diff
@@ -18,6 +18,12 @@ const ERROR_PATTERN = new RegExp('' +
   '(.+?)\\.?$'           // Message text, the ending period is optional for MiKTeX
 )
 
+// Pattern for obtaining the last macro in a line
+// Meant for catching the undefined control sequence
+const LAST_MACRO_PATTERN = new RegExp('' +
+  '[^\\\\]*(\\\\[a-zA-Z@]+)$' // Macro name
```

Comment:
I'm not sure as to what would constitute a legal macro name. 

One could argue that it _might_ be better to just look at the last
space and go from there.

An issue with the definition here is that I din't include underscores `_`, 
which will be very widely used in macro names come LaTeX3.

```suggestion
  '[^\\\\]*(\\\\[a-zA-Z@_]+)$' // Macro name
```